Archiver.php

```diff
@@ -0,0 +1,245 @@
+<?php
+/**
+* User: Zbigniew 'zibi' Jarosik <zibi@nora.pl>
+* Date: 07.05.15 15:19
+ */
+
+namespace IFR\Main;
+
+if( !class_exists('Archiver') )
+{
+	class Archiver
+	{
+		static $_config = [
+			'zip'=>[
+				'compress'=>'zip -r',
```

Comment:
My preferred solution would be to pass this config from application ( atrium in this case ). It might be assembled in packer-specific muyo part ( ideally replacable by php function ) i.e. $MUYO/zip.php or $MUYO/archiver/zip.php, but it's really important to avoid embedding non-obvious (hidden) environment-specific dependencies ( such "zip" program in version which parameters correspond with this code ).
It's a library after all, so minimum of sanity should be preserved.